cmd/der2ascii/writer_test.go

```diff
@@ -131,12 +131,24 @@ var derToASCIITests = []convertFuncTest{
 	// BIT STRINGs are encoded normally if the contents are not an element.
 	{
 		[]byte{0x03, 0x03, 0x00, 0x00, 0x00},
-		"BIT_STRING { `00` `0000` }\n",
+		"BIT_STRING { b`0000000000000000` }\n",
 	},
```

Comment:
I believe you got the bit order correct this time, but let's add some 1, 11, etc., tests now just to be sure. :-)

Reply:
We did get it right, as the new tests confirm. =)